pages/result/result.css

```diff
@@ -0,0 +1,111 @@
+@import "../../css/dafault.css";
+@import "../../css/navbar.css";
+
+.container {
+  margin-bottom: 6rem;
+}
+
+.section-wrapper {
+  min-height: 80vh;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Consider making utility class for these styles as you are using them oftenly. 
```html
  display: flex;
  align-items: center;
  justify-content: center;
```

Reply:
Okay. will do that.